### DISCUSSION TITLE
ReactJS

### DISCUSSION THREAD

**Discussion by MK-Hasan-Mahdi** [MK-Hasan-Mahdi/jspractice#3] [Q&A] [answered]
How components works in React?

**Comment by AyonJD** [comment, marked as answer]
Components are the building blocks of any React app and a typical React app will have many of these. Simply put, a component is a JavaScript class or function that optionally accepts inputs i.e. properties(props) and returns a React element that describes how a section of the UI (User Interface) should appear
